app/processing/statement/components/get-detailed-payments.js

```diff
@@ -0,0 +1,25 @@
+
+const moment = require('moment')
+const { convertToPounds } = require('../../../utility')
+
+const getDetailedPayments = async (calculation, paymentRequest, settlement) => {
+  const payments = []
+
+  const payment = {
+    invoiceNumber: calculation.invoiceNumber,
+    reference: settlement.reference,
+    dueDate: formatDate(paymentRequest.dueDate),
+    settled: formatDate(settlement.settlementDate),
+    calculated: formatDate(calculation.calculated),
+    value: convertToPounds(paymentRequest.value)
+  }
+  payments.push(payment)
+
+  return payments

```

Comment:
Long winded way of just returning the object wrapped in `[]`

Reply:
existing implementation will easily enable dynamic additional of more items when there is need.